docs/custom_check.md

```diff
@@ -1,22 +1,24 @@
-# Custom Check
+# Custom Checks
 
-To configure one of Datadog's 400+ integrations, leverage the [Agent Autodiscovery feature][1]. But if you want to run a custom check, the `DatadogAgent` resource can be configured to provide custom checks (`checks.d`) and their configuration files (`conf.d`) at initialization time. A `ConfigMap` resource needs to be configured for each of these settings before the `DatadogAgent` resource using them is created.
+To run a [custom check][1], you can configure the `DatadogAgent` resource to provide custom checks (`checks.d`) and their corresponding configuration files (`conf.d`) at initialization time. You must configure a ConfigMap resource for each check script file and its configuration file.
 
-Below is an example of configuring these `ConfigMaps` for a single check `hello` that submits the `hello.world` metrics to Datadog. See the [Introduction to Integrations][2] to learn what is a check in the Datadog ecosystem.
+This page explains how to set up a custom check, `hello`, that submits a `hello.world` metric to Datadog.
+
+To learn more about checks in the Datadog ecosystem, see [Introduction to Integrations][2]. To configure a [Datadog integration][3], see [Kubernetes and Integrations][4]
```

Comment:
```suggestion
To learn more about checks in the Datadog ecosystem, see [Introduction to Integrations][2]. To configure a [Datadog Integration][3], see [Kubernetes and Integrations][4]
```
minor nit just wondering if we should be consistent with capitalizing "Integration" ?